docs/UserGuide.md

```diff
@@ -186,8 +187,9 @@ Examples:
 
 Assigns employee(s) to a project in TaskHub
 
-Format: `assignE pr/PROJECT_NAME em/EMPLOYEE_INDEX [em/MORE_EMPLOYEE_INDEX]...`
+Format: `assignE pr/PROJECT_INDEX em/EMPLOYEE_INDEX OTHER_EMPLOYEE_INDEXES...`
```

Comment:
I would say that the project prefix should be present, as this will make it super clear-cut to the user.
For example, intermediate users may get confused on whether to use the `pr/` or the `em/` prefix:
"To assign the 2nd employee to the 2nd project, do I use it like this: `assignE 2 pr/2` or like this: `assignE 2 em/2`?"
However, if we do it like `assignE pr/2 em/2` or `assignE em/2 pr/2`, both works as the ordering does not matter, and does what the user intended, in a clear manner and without getting any "wrong command format" messages.

Doing it like this would help the user avoid certain frustrations.